### DISCUSSION TITLE
Welcome to CSharp Discussions!

### DISCUSSION THREAD

**Discussion by joecare99** [joecare99/CSharp#36] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by klentgwapz** [comment]
https://github.com/klentgwapz/help_me_bisection_method/issues/1#issue-1410154094
Can you help me with this bisection method? I'm stuck at iteration 1 and can't find the another iterations. please help

**Reply by joecare99** [reply]
The iteration in your program ends because the error after the first iteration is -1 ... and -1 IS < 0.000000000000001 (!)
Try: 
` while (Math.Abs(error) > 0.000000000000001)`
